proto-list-final.yaml

```diff
@@ -2409,252 +2395,252 @@ validated: null
 proto: Conversation
-kind: ContactcenterinsightsConversation
+kind: ContactCenterInsightsConversation
 protopath: google.cloud.contactcenterinsights.v1.Conversation
 validated: null
 ---
 service: contactcenterinsights
 package: google.cloud.contactcenterinsights.v1
 proto: Analysis
-kind: ContactcenterinsightsAnalysis
+kind: ContactCenterInsightsAnalysis
 protopath: google.cloud.contactcenterinsights.v1.Analysis
 validated: null
 ---
 service: contactcenterinsights
 package: google.cloud.contactcenterinsights.v1
 proto: FeedbackLabel
-kind: ContactcenterinsightsFeedbackLabel
+kind: ContactCenterInsightsFeedbackLabel
 protopath: google.cloud.contactcenterinsights.v1.FeedbackLabel
 validated: null
 ---
 service: contactcenterinsights
 package: google.cloud.contactcenterinsights.v1
 proto: IssueModel
-kind: ContactcenterinsightsIssueModel
+kind: ContactCenterInsightsIssueModel
 protopath: google.cloud.contactcenterinsights.v1.IssueModel
 validated: null
 ---
 service: contactcenterinsights
 package: google.cloud.contactcenterinsights.v1
 proto: Issue
-kind: ContactcenterinsightsIssue
+kind: ContactCenterInsightsIssue
 protopath: google.cloud.contactcenterinsights.v1.Issue
 validated: null
 ---
 service: contactcenterinsights
 package: google.cloud.contactcenterinsights.v1
 proto: PhraseMatcher
-kind: ContactcenterinsightsPhraseMatcher
+kind: ContactCenterInsightsPhraseMatcher
 protopath: google.cloud.contactcenterinsights.v1.PhraseMatcher
 validated: null
 ---
 service: contactcenterinsights
 package: google.cloud.contactcenterinsights.v1
 proto: Settings
-kind: ContactcenterinsightsSettings
+kind: ContactCenterInsightsSettings
 protopath: google.cloud.contactcenterinsights.v1.Settings
 validated: null
 ---
 service: contactcenterinsights
 package: google.cloud.contactcenterinsights.v1
 proto: AnalysisRule
-kind: ContactcenterinsightsAnalysisRule
+kind: ContactCenterInsightsAnalysisRule
 protopath: google.cloud.contactcenterinsights.v1.AnalysisRule
 validated: null
 ---
 service: contactcenterinsights
 package: google.cloud.contactcenterinsights.v1
 proto: EncryptionSpec
-kind: ContactcenterinsightsEncryptionSpec
+kind: ContactCenterInsightsEncryptionSpec
 protopath: google.cloud.contactcenterinsights.v1.EncryptionSpec
 validated: null
 ---
 service: contactcenterinsights
 package: google.cloud.contactcenterinsights.v1
 proto: View
-kind: ContactcenterinsightsView
+kind: ContactCenterInsightsView
 protopath: google.cloud.contactcenterinsights.v1.View
 validated: null
 ---
 service: contactcenterinsights
 package: google.cloud.contactcenterinsights.v1
 proto: QaQuestion
-kind: ContactcenterinsightsQaQuestion
+kind: ContactCenterInsightsQaQuestion
 protopath: google.cloud.contactcenterinsights.v1.QaQuestion
 validated: null
 ---
 service: contactcenterinsights
 package: google.cloud.contactcenterinsights.v1
 proto: QaScorecard
-kind: ContactcenterinsightsQaScorecard
+kind: ContactCenterInsightsQaScorecard
```

Comment:
Looking at the context, this should be `Qa`: https://cloud.google.com/contact-center/insights/docs/qai-basics#scorecards